docs/import/create_score_asset.js

```diff
@@ -280,28 +280,29 @@ function setMapBoundsInfo(message) {
 function createScoreAssetForStateBasedDisaster(
     disasterData, setMapBoundsInfoFunction = setMapBoundsInfo) {
   setStatus('');
-  const states = disasterData['states'];
-  const assetData = disasterData['asset_data'];
-  const blockGroupPaths = assetData['block_group_asset_paths'];
-  const snapData = assetData['snap_data'];
-  const snapPaths = snapData['paths'];
-  const snapKey = snapData['snap_key'];
-  const totalKey = snapData['total_key'];
-  const sviPaths = assetData['svi_asset_paths'];
-  const sviKey = assetData['svi_key'];
-  const incomePaths = assetData['income_asset_paths'];
-  const incomeKey = assetData['income_key'];
-  // If we switch to CrowdAI data, this will change.
-  const buildingPaths = assetData['building_asset_paths'];
+  const {assetData} = disasterData;
+  const {stateBasedData} = assetData;
+  const {
+    states,
+    blockGroupAssetPaths,
+    snapData,
+    sviAssetPaths,
+    sviKey,
+    incomeAssetPaths,
+    incomeKey,
+    buildingAssetPaths,
+  } = stateBasedData;
+  const {paths: snapPaths, snapKey, totalKey} = snapData;
+  // If we switch to CrowdAI data, building asset paths will change.
```

Comment:
This means the structure will change not the value right?

Reply:
Removed. I think this is taken care of by the TODO below about filtering buildings out from damage, and Hans seems happy with never using CrowdAI buildings data.